IFComp/lib/IFComp/Form/Entry.pm

```diff
@@ -27,6 +27,66 @@ has_field 'subtitle' => (
     maxlength => 128,
 );
 
+has_field 'platform' => (
+    type         => 'Select',
+    id           => 'platform',
+    label        => 'Gameplay platform',
+    options      => [
+        {
+            value => 'website',
+            label => 'Browser-based JavaScript (including Twine)',
+        },
+        {
+            value => 'inform',
+            label => 'Glulx or Z-code, not packaged with a website',
+        },
+        {
+            value => 'quixe',
+            label => q{Glulx, using Inform's "Release with a website" feature},
+        },
+        {
+            value => 'parchment',
+            label => q{Z-code, using Inform's "Release with a website" feature},
+        },
+        {
+            value => 'inform-website',
+            label => 'Glulx or Z-code, presented within a custom website',
+        },
+        {
+            value => 'quest',
+            label => 'Quest (download-only)',
+        },
+        {
+            value => 'quest-website',
+            label => 'Quest (online play available)',
+        },
+        {
+            value => 'tads',
```

Comment:
I suggest you split this up into TADS 3 and TADS 3 Web UI, as handling them will probably be different.

Reply:
This is a good idea, but my goal here is to keep the list of possible platforms exactly as it was before this change. We could introduce new possible platforms (and rules about handling them) as a future step.

Reply:
They are substantially different platforms, and people are discussing them on the forum now. I'm not sure if Web UI games have ever been submitted before, but they really shouldn't be lumped together. So don't forget to do the follow up step ;)